package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.5.1",
     "@toruslabs/torus-embed": "1.9.2",
-    "@types/node-fetch": "^2.5.8",
+    "@types/node-fetch": "^2.5.10",
```

Comment:
why are we using node fetch?

Reply:
We were using it for easier fetch with the server in friendly. We were planning to use it to do the same with the example client server. However it currently isn't being used. I'll remove it for now, and just add it back when we need it.